cagallo/book-club-api
# Feature/deployment to heroku ect

## What Features are Being Added?
- Created `.env` 
- Set up production environment in `knexfile.js`
- Installed dotenv npm package
- Added script to run server in package.json

## What is Being Refactored?
- Refactored repo structure. 
  - Removed db repo to solve bug with heroku being unable to find migrations and seeds directories
- Changed environment connection from development to production in `queries.js`

## Where should the reviewer start?
- Start in knexfile.js to see the new production environment. 

## How should this be manually tested?
- Using our deployed link: `https://book-club-api-2110.herokuapp.com/api/v1/` try out the different end points on postman to see whether we are getting back the proper responses/error handling messages.

## What are the relevant tickets?
- Deploy to heroku!
## Any Other Notes/Questions/Problems?
- Way to go everyone, that was not easy and I'm proud of the way we all stuck through it and figured this out!
